content/programming-cookbook/logging/NLog.md

```diff
@@ -0,0 +1,66 @@
+---
+title: NLog
+showMetadata: true
+editable: true
+showToc: true
+order: 3
+---
+
+# How to setup NLog
+
+- Create a new .NET Framework console project. This can be applied to .NET Core
+- Install NLog Nuget package.
+```
+install-package NLog
+
+```
+
+- Edit NLog.config file
+```
+    <?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Very minor, I think for XML we can make 2 space indention, could you please update this XML code to use 2 indention.